src/app/routines/publicRoutines/routineListItem/RoutineListItem.scss

```diff
@@ -19,4 +20,40 @@
     line-clamp: 2;
     -webkit-line-clamp: 2;
   }
+
+  &__likes {
+    width: 4.5rem;
+    height: 1.5rem;
+    display: flex;
+    border-radius: $border-radius-medium;
+    border: 1px solid $secondary-color;
+    overflow: hidden;
+
+    & .active {
+      color: $accent-color;
+    }
+    &:hover {
+      cursor: pointer;
+    }
+  }
+
+  &__likes-number {
+    width: 100%;
+    max-width: 2.25rem;
+    padding-left: 0.35rem;
+    padding-right: 0.2rem;
+    background-color: $secondary-color;
+    overflow: hidden;
+
+    & p {
+      height: 100%;
+      text-align: end;
+      vertical-align: middle;
+    }
+  }
+
+  & .likes-icon {
+    padding: 0.15rem 0.25rem 0.15rem 0.15rem;
+    border-left: 1px solid $secondary-color;
```

Comment:
same

Reply:
same thing it was like this in the other component